src/scenes/home/benefit/benefit.css

```diff
@@ -25,22 +25,33 @@
   font-size: 0.8em;
 }
 
-.nodeLogo {
-  max-width: 25vw;
+.logo {
+  max-width: 25%;
   height: auto;
-  margin: 3vw;
+  margin: 2vw;
 }
-
-.codeFellowsLogo {
-  max-width: 30vw;
+.pvaLogo {
+  max-width: 25%;
   height: auto;
-  margin: 3vw;
+  margin: 2vw;
 }
-
-.sabioLogo {
-  margin: 3vw;
+.srvdLogo {
+  height:auto;
+  max-width: 10%;
+  margin: 2vw;
 }
 
```

Comment:
There are some duplicated CSS rules here among the `.logo`, `.pvaLogo`, and `.srvdLogo` classes - they all have the same `height` and `margin` rules - these could probably be in one class (`.logo`) and applied to all of the logo elements, and the other classes (`.srvdLogo`, `.pvaLogo`) could be applied only to those elements where values need to differ (looks like `max-width` is the only unique property here.